specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/sensitivityLabels.json

```diff
@@ -86,6 +86,13 @@
           {
             "$ref": "#/parameters/DatabaseNameParameter"
           },
+          {
+            "name": "showDisabledRecommendations",
```

Comment:
"show" is UI specific. Perhaps "include" or "return" instead

Reply:
Updated